docs/improving-gci-experience.md

```diff
@@ -0,0 +1,377 @@
+As such it's open for further suggestions and improvements, to learn the
+most from diverse experiences.
+
+It also includes answers to many common questions and issues raised during
+code reviews and discussions on Zulip.
+[Social rules](#what-are-the-social-rules), [GCI tasks](#how-gci-tasks-work),
+and some ideas on [going beyond GCI](#how-do-i-go-beyond-gci-tasks)
+have been discussed here.
+
+# What are the social rules?
+
+As a Zulip contributor, you should adhere to our [community code of
+conduct](https://zulip.readthedocs.io/en/latest/code-of-conduct.html).
+There are also some GCI lessons learned:
+
+* Respect time and effort others are investing in their contributions and
+communication.
+
+* Be mindful of others' time - fellow participants are also waiting for their
+reviews in the task queue and mentors are managing multiple responsibilities
+while online.
+
+* Never assume malice - if you are waiting for review or an answer, it's
+usually because the other person is busy with their work, unavailable or
+waiting for you to answer their questions.
+
+* Be mindful of cultural differences - both participants and mentors come
+from different backgrounds (including countries, languages and cultures),
+please remember some nuances might be interpreted differently than you'd
+expect.
+
+* Please don't at-mention people - all of the mentors are reading through the
+feed as quickly as they can, while helping people.
+
+If you are new to the concept of the netiquette, please read this [helpful
+resource](http://www.albion.com/netiquette/).
+
+# How GCI tasks work?
+
+GCI participants are required to complete tasks that have been published in
+the GCI app. In order to start working on a task, a student has to claim it
+in the app. When they've finished the task, they need to submit it in the
+app for review. A student can have only one active (claimed or submitted)
+task at a time.
+
+### Is GCI all about the number of tasks?
+
+In the [GCI
+rules](https://developers.google.com/open-source/gci/resources/contest-rules)
+there are two important factors for choosing finalists and grand-prize
+winners.
+
+* Number of tasks
+  > Each complete task entry is worth one (1) point.
+
+  In order to be considered for the grand-prize, a student needs to be in
+  the top-10 in terms of number of tasks.
+
+* Other factors
+  > Each entry will be judged according to the following factors: creativity,
+  thoroughness, and quality of work. The Organization may additionally factor
+  the contestant’s involvement in the Organization’s community in the judging.  
+
+While the number of tasks that they complete is important, the winning
+students are often chosen for not only the number of tasks that they
+completed but also their willingness to help other students and participate
+in the community. Thus being in the top-10 is a requirement, but the
+student with the most completed tasks won't necessarily win the grand-prize.
```

Comment:
Might add something specific like "fixing an issue in the project that happens to not have a GCI task is worth just as much as completing a GCI task for that issue"